core/plugins/org.polarsys.capella.core.data.core.properties/src/org/polarsys/capella/core/data/core/properties/filters/LegacyTabFilter.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *  
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.core.data.core.properties.filters;
+
+import static java.util.Arrays.asList;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.eclipse.eef.properties.ui.api.IEEFTabDescriptor;
+import org.eclipse.eef.properties.ui.api.IEEFTabDescriptorFilter;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.gef.EditPart;
+import org.eclipse.gmf.runtime.notation.View;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.jface.viewers.StructuredSelection;
+import org.eclipse.sirius.viewpoint.DSemanticDecorator;
+import org.eclipse.sirius.viewpoint.description.DAnnotation;
+import org.eclipse.ui.IWorkbenchPart;
+/**
+ * Used to filter out Sirius legacy tabs from the Properties view
+ * when the selection is a custom model.
+ */
+public class LegacyTabFilter  implements IEEFTabDescriptorFilter {
+  
+  /** IDs of the pages to filter out */
+  private static final Set<String> TABS_TO_FILTER_OUT = new HashSet<>(
+      asList("property.tab.semantic")

```

Comment:
use a constant for "property.tab.semantic"

Reply:
I think is ok to keep the set also, for now is only one tab, but to keep it extensible.